src/config/ini.c

```diff
@@ -13,7 +13,7 @@ home page for more info:
 
 #include "ini.h"
 
-#define MAX_LINE 200
+#define MAX_ONELINE 200
 #define MAX_SECTION 50
```

Comment:
I don't love this name.  How about MAX_INI_LINE_LENGTH

Reply:
okay